### PR TITLE
[#2089] Remove UI-Kit dependency in editor/module.js

### PR DESCRIPTION
**Pull request by cadecairos**
Lighthouse Ticket:  https://webmademovies.lighthouseapp.com/projects/65733/tickets/2089-crufty-dependency-in-define-call-in-srceditormodulejs

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/289a6bb94b5d91a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/7bbd867bcc320a9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/289a6bb94b5d91a/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/7bbd867bcc320a9/output.txt

Total script time: 3.00 mins

File: src/ui/ui.js [hunk 1]
```diff
@@ -327,10 +327,16 @@ define( [ "core/eventmanager", "./toggler",
...
+    // This is an easter egg to open a UI kit editor. Hurrah
+    _this.showUIKit = function() {
+      butter.editor.openEditor( "ui-kit" );
+    };
+
   } //UI
 
   UI.__moduleName = "ui";
 
+
```
Comment (mjschranz) [inline src/ui/ui.js]: Remove this extra new line :)

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f594b0c5b83a1b3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/65ffd8b60572ec3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/f594b0c5b83a1b3/output.txt

Total script time: 2.19 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/65ffd8b60572ec3/output.txt

Total script time: 1.14 mins

Server is running on http://pr1020.butyr.org/templates/basic/
Run the Test Suite: http://pr1020.butyr.org/test/

**Pull request merged**